data/hai/hai outfits.txt

```diff
@@ -681,7 +681,7 @@ outfit `"Basrem" Reverse Thruster`
 	"reverse flare sprite" "effect/atomic flare/tiny"
 		"frame rate" 14
 	"reverse flare sound" "atomic tiny"
-	description "These tiny reverse thrusters allow even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
+	description "These tiny Reverse Thrusters allow even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

Comment:
```suggestion
	description "These tiny reverse thrusters allow even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

"Reverse Thruster(s)" on their own is not an outfit name being referred to here. None of these three Hai reverse thruster descriptions actually use the name of the outfit but instead refer to "reverse thrusters" as a general thing.

Alternative fix would be to reword each slightly to have their own full outfit name and capitalize that.